other/test-runner/index.js

```diff
@@ -83,24 +87,24 @@ function main() {
 }
 
 function process_args(config) {
-  if(config.args.browser) {
+  if (config.args.browser) {
```

Comment:
For the future, it would have been ideal to make these whitespace changes in a separate pull request.